src/components/media-player/ha-media-player-browse.ts

```diff
@@ -726,10 +728,14 @@ export class HaMediaPlayerBrowse extends LitElement {
           background-color: transparent;
         }
 
-        mwc-list-item .title {
+        mwc-list-item[dir="ltr"] .title {
           margin-left: 16px;
         }
 
+        mwc-list-item[dir="rtl"] .title {
+          margin-right: 16px;
+        }
```

Comment:
```suggestion
        mwc-list-item .title {
          margin-left: 16px;
        }

        mwc-list-item[dir="rtl"] .title {
          margin-right: 16px;
          margin-left: 0;
        }
```

I think this is faster. Attributes are slow :)

Reply:
Will push a commit. Github somehow tells me your proposal is outdated, so I cannot accept it here directly.

Reply:
Do we set the dir attribute for this element?

Reply:
He added it yes